content/docs/installation/scala.md

```diff
@@ -1,11 +1,51 @@
 ---
 title: Cucumber-Scala
 subtitle: Scala
-img: /img/scala.png
+svg: installation/scala.svg
 implementation: unmaintained
 weight: 1315
 ---
 
+----
+
 **This package is currently in need of new maintainers.**
 
 Please see [Cucumber-Scala](https://github.com/cucumber/cucumber-jvm-scala).
+
+----
+
+Cucumber-Scala is published in the central Maven repository.
+You can install it by adding dependencies to your project.
+
+{{% note "Dependencies"%}}
+Make sure the Cucumber version is the same for all Cucumber dependencies.
+{{% /note %}}
+
+# Maven
+
+```xml
+<dependency>
+    <groupId>io.cucumber</groupId>
+    <artifactId>cucumber-scala_2.13</artifactId>
+    <version>{{% version "cucumberjvm" %}}</version>
+    <scope>test</scope>
+</dependency>
+```
+
+You can now run Cucumber [from the command line](/docs/cucumber/api/#from-the-command-line) or [run Cucumber with Maven](/docs/tools/java#maven).
+
+# Sbt
```

Comment:
What is "Sbt"? Or would everyone using Scala know?

Reply:
It's one of the most used build tool for Scala (roughly replaces Maven or Gradle). I think everyone who uses Scala knows about sbt.